modules/renter/downloadcache.go

```diff
@@ -41,7 +53,8 @@ func (r *Renter) managedTryCache(udc *unfinishedDownloadChunk) bool {
 	udc.mu.Lock()
 	defer udc.mu.Unlock()
 	r.cmu.Lock()
-	data, cached := r.chunkCache[udc.staticCacheID]
+	cd, cached := r.chunkCache[udc.staticCacheID]
+	data := cd.data
```

Comment:
Instead of assigning `cd.data` to `data` you could simply replace the following occurences of `data` with `cd.data`.